src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets

```diff
@@ -32,6 +32,7 @@
     <EmccExportedFunction Include="stackAlloc" />
     <EmccExportedFunction Include="stackRestore" />
     <EmccExportedFunction Include="stackSave" />
+    <EmccExportedFunction Include="_minipal_get_non_cryptographically_secure_random_bytes" />
```

Comment:
Do you know what uses it / why it needs to be exported?

Reply:
Hmm, funny, its not needed so removed it.